AutoUpdate.py

```diff
@@ -0,0 +1,34 @@
+#/usr/bin/env python
```

Comment:
If I simply run "python AutoUpdate.py" it does a git clone or pull (as intended) but then simply dumps me in a Python shell. This should probably test for the right number (and type?) of arguments and give a proper error/usage message.